Automattic/wp-calypso
# GF Signup : Fix post purchase upsells for paid media flow

Partially Fixes - Automattic/martech#1882

## Proposed Changes

- Since the redirect_to parameter was set as a query param for checkout. The inbuilt cart based upsells were not being activated.
- This change moves this url to the destination cookie so that eventually this is picked up by the upsell nudge component. 


## Testing Instructions
* Go through the `/setup/onboarding-media` flow 
* Purchase a premium plan
* Complete checkout
* Make sure that you are redirected to the business plan upsell

Also try any other upsell that you are familiar with on the `/start` onboarding. That is if you are familiar with such an upsell.


## Pre-merge Checklist
